lbry/testcase.py

```diff
@@ -484,13 +484,15 @@ def sout(self, value):
         """ Synchronous version of `out` method. """
         return json.loads(jsonrpc_dumps_pretty(value, ledger=self.ledger))['result']
 
-    async def confirm_and_render(self, awaitable, confirm) -> Transaction:
+    async def confirm_and_render(self, awaitable, confirm, return_tx=False) -> Transaction:
         tx = await awaitable
         if confirm:
             await self.ledger.wait(tx)
             await self.generate(1)
             await self.ledger.wait(tx, self.blockchain.block_expected)
-        return self.sout(tx)
+        if not return_tx:
+            return self.sout(tx)
+        return tx
```

Comment:
This should be split into two methods. One that confirms and renders and another one that just confirms (the confirm_and_render would call the confirm only one). Otherwise you have a single method with different return types.